### PR TITLE
Added `data-testid`s to UI elements

### PR DESCRIPTION
**Pull request by tplevko**
Fixes: [#1890](https://github.com/atlasmap/atlasmap/issues/1890)
This should simplify the rewrite of test suite from Angular UI to the new React UI

### CODE REVIEW

File: ui-react/packages/atlasmap-ui/src/common/useSingleInputDialog.tsx [hunk 1]
```diff
@@ -74,16 +74,23 @@ export function useSingleInputDialog({
...
           isRequired={true}
           isValid={isValid}
+          data-testid={'input-text-field'}
         />,
         <Button
           key={'confirm'}
           variant={'primary'}
           onClick={handleConfirm}
           isDisabled={!isValid}
+          data-testid={'confirm-button'}
```
Comment (gashcrumb) [inline ui-react/packages/atlasmap-ui/src/common/useSingleInputDialog.tsx]: Is that specific enough to be unique on the page?

File: ui-react/packages/atlasmap-ui/src/common/useSingleInputDialog.tsx [hunk 2]
```diff
@@ -74,16 +74,23 @@ export function useSingleInputDialog({
...
         >
           Confirm
         </Button>,
-        <Button key={'cancel'} variant={'link'} onClick={handleCancel}>
+        <Button
+          key={'cancel'}
+          variant={'link'}
+          onClick={handleCancel}
+          data-testid={'cancel-button'}
```
Comment (gashcrumb) [inline ui-react/packages/atlasmap-ui/src/common/useSingleInputDialog.tsx]: Same with the above.

Comment (tplevko) [reply ui-react/packages/atlasmap-ui/src/common/useSingleInputDialog.tsx]: sure, makes sense. Updated all the 'cancel-button' and 'confirm-button' test-id's

**Comment by igarashitm** [comment]
@tplevko seems it got some conflict against master, could you rebase? @atlasmap/ui is this good to go now?

**Comment by riccardo-forina** [comment]
Yep

**Comment by tplevko** [comment]
@igarashitm sure thing! Should be all right now.

**Pull request merged**